doc/users/installing/index.rst

```diff
@@ -291,8 +291,6 @@ from the Terminal.app command line::
 
    python3 -m pip install matplotlib
 
-(``sudo python3.6 ...`` on Macports).
```

Comment:
```
(``port install py-matplotlib`` on Macports).
```
seems like a correct replacement.

https://apple.stackexchange.com/questions/282839/using-pip-vs-macports-for-installing-python-packages
https://ports.macports.org/port/py-matplotlib/details/

Reply:
Should we just drop this altogether?  We don't give instructions for apt or other package managers, do we?  I think we should doc pip and conda and if someone wants to install with macports they can ask macports how to do it.

Reply:
We give basic installation commands (like `sudo apt install python-matplotlib`) for some package managers, but little package-manager specific setup instructions.

https://github.com/matplotlib/matplotlib/blob/main/doc/users/installing/index.rst#linux-package-manager

This does seem to be a common problem for OSX users, so maybe it might be worth leaving in the installation guide and adding @oscargus suggestion.

Reply:
I somewhat disagree, or at the least we need to rephrase the docs here.   If I was a naive user who had not really heard of conda I'd skip the conda section (what's that?) and go to the MacOS section.  Then I would be using crazy macports or brew installs, or worse trying to use the system python, when really by __far__ the easiest solution on a Mac (or most other platforms) is to use conda.    All this stuff should be hidden behind links that say "manual OS-specific installs, not recommended".